### ISSUE TITLE
Renaming hologram in config

### ISSUE THREAD

**Issue by tacticalNukes** [filoghost/HolographicDisplays#581]
### Confirmation

- [X] I have read the [FAQ](https://filoghost.me/docs/holographic-displays/faq).
- [X] I have tested the [latest development build](https://ci.codemc.io/job/filoghost/job/HolographicDisplays) of Holographic Displays and the bug is still present.
- [X] I have updated **Spigot** to the latest release for my particular Minecraft version.
- [X] I made sure the bug hasn't already been reported on the [issue tracker](https://github.com/filoghost/HolographicDisplays/issues?q=is%3Aissue).

### Description

So basically I edited a hologram in config and changed it's name, now it's displaying the old "default hologram" text and the new one I added.

### How to reproduce

1. Create a hologram
2. Change the name in config
3. and reload

### Server version

1.17.1 paper

### Holographic Displays version

3.0.0-snapshot

**Comment by filoghost** [comment]
I'm not able to reproduce the issue. Please show the configuration before and after editiing it.

**Comment by tacticalNukes** [comment]
So I found that the problem is that it's not deleting old hologram text, so if I would in my example change a holograms text the old one would stay there and the new one would be put over it, they are then overlapping eachother as like i've created 2 different holograms inside eachother. This gets fixed once I relog on my server.

**Comment by filoghost** [comment]
Still not able to reproduce it using the latest dev build.
